pkg/apis/bundle/v1alpha1/bundle_types.go

```diff
@@ -55,12 +49,16 @@ type File struct {
 
 // ComponentPackageSpec represents the spec for the component.
 type ComponentPackageSpec struct {
-	// Version-string for this component. The version should be a SemVer 2 string
-	// (see https://semver.org/) of the form X.Y.Z (Major.Minor.Patch).  A
-	// major-version changes should indicate breaking changes, minor-versions
-	// should indicate backwards compatible features, and patch changes should
-	// indicate backwards compatible. If there are any changes to the component,
-	// then the version string must be incremented.
+	// CanonicalName is the canonical name of this component. For example, 'etcd' or
+	// 'kube-proxy'.
+	CanonicalName string `json:"canonicalName,omitempty"`
```

Comment:
Are we going to generate protos for these objects, or does CRD not yet have proto support (once you start generating protos there's some extra stuff that goes in the struct tags)?

Reply:
The plan is to generate protos, yeah. Just haven't done it yet.